cli/lsp/tsc.rs

```diff
@@ -231,9 +231,15 @@ impl TsServer {
             start(&mut ts_runtime, false).unwrap();
             started = true;
           }
-          let value = request(&mut ts_runtime, state_snapshot, req, token);
+          let value =
+            request(&mut ts_runtime, state_snapshot, req, token.clone());
+          // Don't print the send error if the token is cancelled, it's expected
+          // to fail in that case and this commonly occurs.
+          #[allow(clippy::collapsible_if)]
```

Comment:
See https://github.com/denoland/deno/pull/21387#discussion_r1410104323 for getting rid of clippy